internal/slack/slack.go

```diff
@@ -313,6 +313,9 @@ func (c *Client) NotifyK8SPodErrorEvent(ctx context.Context, event *http.PodErro
 	if err != nil {
 		return err
 	}
+	if event.AlertSquad != "false" {
```

Comment:
Error check missing?

Reply:
It is a draft still :P

Reply:
Haha, oh sorry 🙈

Reply:
I'm not sure the error check is needed after :)
This
```
	if err != nil {
		return err
	}
	return err
```
does the same as at the end of the function 
```
return err
```

Reply:
I guess the next "feature gate" for aura and odyssey should be removed with this addition?

Currently we "could" override the `err` if a channel is set _and_ it is squad aura :)

Reply:
Just saw that it was removed 🙈

Reply:
It has been removed :) Then we would have our squad hardcoded into a public repo